x/environment.py

```diff
@@ -60,10 +60,15 @@ class Catcher(Enviroment):
     grid_size : int
         Size of the square grid
     """
-    def __init__(self, grid_size=10, output_shape=None):
+    def __init__(self, grid_size=10, output_shape=None, output_type='pixels'):
```

Comment:
Table couldn't possibly learn from pixels. The dimensionality is too high. To train it we need a lower dimensional representation. I have it give the x,y position of the ball and the x position of  the paddle